src/square.cc

```diff
@@ -380,6 +380,18 @@ ModelInstanceState::RequestThread(
       TRITONBACKEND_ResponseFactory, backend::ResponseFactoryDeleter>
       factory(factory_ptr);
 
+  if (element_count == 0) {
```

Comment:
Why did we have to move this block on top? 
Shouldn't the logic be skipped for `element_count==0`? Sending such flags looks cleaner towards the end.

Reply:
Updated.